.travis.yml

```diff
@@ -29,19 +22,33 @@ matrix:
     fast_finish: true
     include:
         - php: 5.4
-          env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-ci" SYMFONY_VERSION=2.7.*
+          env: DEPENDENCIES=minimum COVERAGE=true TEST_COMMAND="composer test-ci" SYMFONY_VERSION=2.7.*
+        - php: 5.5
+        - php: 5.6
+        - php: 7.0
+        - php: 7.1
+          env: SYMFONY_VERSION=3.0.*
```

Comment:
Why `3.0.*`? Didn't you mean `3.4.*` instead?

Reply:
I kept this requirements from the selection above. Something like `env: DEPENDENCIES="symfony/lts:^3"` probably makes more sense?

Reply:
I would also update the other places where `symfony/symfony` is used by using `symfony/lts` instead. That way you won't hide bugs related to not installed components outside the CI environment.

Reply:
i guess best see https://github.com/symfony/symfony-docs/pull/8701/ - or lets just make things work here and then sync to that when the recommendation for a perfect .travis.yml has been merged.